daemon/src/position_metrics.rs

```diff
@@ -0,0 +1,344 @@
+use crate::db;
+use async_trait::async_trait;
+use futures::StreamExt;
+use model::CfdEvent;
+use model::EventKind;
+use model::OrderId;
+use model::Position;
+use model::Usd;
+use rust_decimal::prelude::FromPrimitive;
+use rust_decimal::Decimal;
+use std::time::Duration;
+use tokio_tasks::Tasks;
+use xtra_productivity::xtra_productivity;
+use xtras::SendInterval;
+
+// TODO: ideally this would be more often
+pub const UPDATE_METRIC_INTERVAL_SECONDS: Duration = Duration::from_secs(60);
```

Comment:
```suggestion
pub const UPDATE_METRIC_INTERVAL: Duration = Duration::from_secs(60);
```
(and roll-out where used)

nit: it's already a `Duration`, so it's not in "seconds"